### PR TITLE
reduce action space to 4

### PR DESCRIPTION
**Pull request by taufeeque9**

### CODE REVIEW

File: envpool/sokoban/sokoban_envpool.cc [hunk 1]
```diff
@@ -75,13 +75,8 @@ void SokobanEnv::Step(const Action& action_dict) {
...
 
   const int action = action_dict["action"_];
-  if (action == kActNoop) {
-    WriteState(static_cast<float>(reward_step_));
-    return;
-  }
-  // From here on, assume the agent will try to move
 
-  const int change_coordinates_idx = (action - 1) % kChangeCoordinates.size();
+  const int change_coordinates_idx = (action) % kChangeCoordinates.size();
```
Comment (rhaps0dy) [inline envpool/sokoban/sokoban_envpool.cc]:
```suggestion
  const int change_coordinates_idx = action;
```

**Review by rhaps0dy — APPROVED** [review]
LGTM thank you!

**Comment by rhaps0dy** [comment]
Please also merge #8 if it hasn't merged yet :(

File: envpool/sokoban/sokoban_envpool.cc (hunk 1 above)
Comment (rhaps0dy) [reply envpool/sokoban/sokoban_envpool.cc]: Action should always be less than 4. If it's not, the `.at()` below should error and that's what we want.

Comment (taufeeque9) [reply envpool/sokoban/sokoban_envpool.cc]: yeah, I wanted to make that change but thought it was okay to leave it.

I'll make that change now!

**Pull request merged**